package.json

```diff
@@ -115,23 +115,24 @@
     "@babel/preset-typescript": "^7.18.6",
     "@babel/register": "^7.18.9",
     "@percy/cli": "^1.20.0",
-    "@percy/storybook": "^4.3.5",
+    "@percy/storybook": "^4.3.6",
     "@skyscanner/bpk-foundations-web": "^16.1.0",
     "@skyscanner/bpk-svgs": "^16.4.0",
-    "@skyscanner/stylelint-config-skyscanner": "^7.0.0",
-    "@storybook/addon-a11y": "^6.5.14",
-    "@storybook/addon-actions": "^6.5.14",
-    "@storybook/addon-viewport": "^6.5.14",
-    "@storybook/builder-webpack5": "^6.5.16",
-    "@storybook/manager-webpack5": "^6.5.16",
-    "@storybook/react": "^6.5.14",
+    "@skyscanner/stylelint-config-skyscanner": "^10.0.0",
+    "@storybook/addon-a11y": "^7.1.1",
+    "@storybook/addon-actions": "^7.1.1",
+    "@storybook/addon-docs": "^7.1.1",
+    "@storybook/addon-viewport": "^7.1.1",
+    "@storybook/react": "^7.1.1",
+    "@storybook/react-webpack5": "^7.1.1",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^12.1.0",
     "@testing-library/user-event": "^14.2.0",
     "@types/jest": "^29.5.2",
     "@types/jest-axe": "^3.5.3",
     "@types/react": "^17.0.37",
     "@types/react-dom": "^17.0.11",
+    "@types/webpack-env": "^1.18.1",
```

Comment:
Due to some dependecies of Storybook having been updated with the v7 migration, we need this [here](https://github.com/Skyscanner/backpack/blob/main/packages/bpk-component-icon/all.tsx#L32)